letsencrypt/tests/auth_handler_test.py

```diff
@@ -167,10 +215,12 @@ def setUp(self):
         from letsencrypt.auth_handler import challb_to_achall
         from letsencrypt.auth_handler import AuthHandler
 
-        # Account and network are mocked...
+        # Account, config and network are mocked
         self.mock_net = mock.MagicMock()
+        self.config = mock.MagicMock(
```

Comment:
ah, this is actually a line within a different `setUp`. Okay, so I have the same comment as in https://github.com/letsencrypt/letsencrypt/pull/1876/files#r49932087. There is no point in specifying port values, just use `self.config = mock.Mock()`